pitch/auto-transpose/README.md

```diff
@@ -22,19 +21,13 @@ So the following displays a C major scale, which sounds like a B flat major scal
 }
 ```
 
-If the instrument is switched, the auto-transpose will follow, and the key signature will be reprinted automatically. The inserted key signatures are printed by a separate engraver.
+If the instrument is switched, the auto-transpose will follow, and the key signature will be reprinted automatically.
 
 Known issue:
 
-* If a key change happens at the same moment as a transposition change, printing the correct key signature depends on the order in which the Lilypond commands are written. See the examples in auto-transpose-keys-1.ly and auto-transpose-keys-2.ly for details.
+* If a key change happens at the same moment as a transposition change, a warning may be triggered ``warning: conflict with event: `key-change-event'``. These can be safely ignored.
 
 There are some TODOs:
 
 * If the music is given in instrument pitch and shall be displayed, the `instrumentTransposition` property still has to be set, but that leads to incorrect MIDI-pitch.
```

Comment:
"shall be displayed" is not very clear, can you rewrite? Even better, provide a workaround for this issue (e.g. wrapping in \transpose).

Reply:
Rewritten in the updated Readme. Suggested workaround is to use a separate \score block for MIDI. The issue was present in the original auto-transpose.

Reply:
This is excellent, thanks.